docker-hub/download-rate-limit.md

```diff
@@ -50,6 +50,7 @@ Valid manifest API requests to Hub will usually include the following rate limit
 ```
 ratelimit-limit    
 ratelimit-remaining
+docker-RateLimit-Source
```

Comment:
Are they CamelCased in the actual response, or lowercase? (looks inconsistent; also I see the example below use lowercase)

Reply:
it should be all lowercase.

Reply:
Yes modifying to lowercase :+1: